Library/ViewModels/ProjectEnvironmentalCommitmentCellViewModel.swift

```diff
@@ -42,3 +45,20 @@ public final class ProjectEnvironmentalCommitmentCellViewModel:
   public var inputs: ProjectEnvironmentalCommitmentCellViewModelInputs { self }
   public var outputs: ProjectEnvironmentalCommitmentCellViewModelOutputs { self }
 }
+
+private func internationalizedString(for category: ProjectCommitmentCategory) -> String {
+  switch category {
+  case .longLastingDesign:
+    return Strings.Long_lasting_design()
+  case .sustainableMaterials:
+    return Strings.Sustainable_materials()
+  case .environmentallyFriendlyFactories:
+    return Strings.Environmentally_friendly_factories()
+  case .sustainableDistribution:
+    return Strings.Sustainable_distribution()
+  case .reusabilityAndRecyclability:
+    return Strings.Reusability_and_recyclability()
+  case .somethingElse:
+    return ""
```

Comment:
Are we still missing this string?

Reply:
Did not see this come back from the script when I ran it. Will include it into the TODO section above.

Reply:
No deploy as of right now so this will have to wait until next week. Everything in the TODO is accurate but will probably depend on the web deploy + translations being complete beforehand.